Week3/homework/step2-7.js

```diff
@@ -20,4 +20,12 @@ f2(y);
 
 console.log(y);
 
-// Add your explanation as a comment here
+// The first function incriminate variable of val 1 number.
+// Then returns the number as a variable.
+// In the console it is written 9, because x was a const variable.
+// That means x never changes.
```

Comment:
Almost there, exactly as you said:
> The first function incriminate variable of val 1 number, Then returns the number as a variable.

And I will add from here on, that in the console it is written as 9, because what happened here that we passed a ver new value to the function as a `val` name, so the `val` parameter contains a very fresh value and
> what happened in Vegas stays in Vegas.

to explain the `const` thing, you can change it to be `let` or even `var`, that will not change it's value unless you explicitly specified:
```javascript
// assuming x declared using `var` or `let`
x = "something else"
```

then `x` is `something else`, in the `const` scenario this will throw an error, and that's the result of a constant, if you try to change it, it will complain and it will not change.